### DISCUSSION TITLE
Sonoff 4CH - every 3 seconds tele information on log

### DISCUSSION THREAD

**Discussion by HarryHase4** [arendst/Tasmota#21463] [Support request]
Tasmota 14.0.0 (release-tasmota) fresh installation with reset configuration, only device is set to sonoff 4ch.
console logging AND mqtt log for ch4 every 3 seconds (tele period is default 300). 

```
15:05:06.653 RSL: RESULT = {"POWER4":"OFF"}
15:05:06.656 RSL: POWER4 = OFF
15:05:09.863 RSL: RESULT = {"POWER4":"OFF"}
15:05:09.866 RSL: POWER4 = OFF
15:05:11.939 RSL: RESULT = {"POWER4":"OFF"}
15:05:11.942 RSL: POWER4 = OFF

```

tele period change has no effect

![Screenshot 2024-05-21 161834](https://github.com/arendst/Tasmota/assets/104375025/5594389f-2b2b-4d05-a928-fc35011ea92c)

**Comment by HarryHase4** [comment]
3 from 45 devices have such a behaviour (sonoff 4 channel - every 3 seconds ; sonoff-touch-switch - 1 minute ; relais-device - 1 minute)

**Reply by sfromis** [reply]
To see what type of event is triggering those actions, enable `Weblog 3` first.

How your gpio config exactly is can also be relevant.

**Reply by HarryHase4** [reply]
weblog 3 was the missing hint ... multiple issues on my side ..... there are old rules which was sending the status and a old script which was asking for device status .. reset 1 was not enough!
